docs/Versions.asciidoc

```diff
@@ -73,3 +73,11 @@ Shared attribute values are pulled from elastic/docs
 ///////
 
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+///////
+APM does not build n.x documentation. Links from .x branches should point to master instead
+///////
+ifeval::["{source_branch}"=="7.x"]
+:apm-server-ref:       {apm-server-ref-m}
```

Comment:
I think this is the issue. The later references use `apm-server-ref-v`.
```suggestion
:apm-server-ref:       {apm-server-ref-m}
:apm-server-ref-v:       {apm-server-ref-m}
```

Reply:
🤦 🙏